src/common/services/Helpik/Helpik.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+
+import Popover from "react-bootstrap/Popover";
+import Overlay from "react-bootstrap/Overlay";
+
+import * as API from "api/http/helpik";
+
+import OutsideAlerter from "./components/OutsideAlerter";
+
+export default class Helpik extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            helpikStatus: false,
+            helpikContent: "",
+            helpikURL: "",
+            helpikVisability: "d-none",
+            target: null,
+        };
+        this.overlayRef = React.createRef();
+
+        this.handleIconClick = this.handleIconClick.bind(this);
+        this.OutsideAlerterCallback = this.OutsideAlerterCallback.bind(this);
+    }
+
+    componentDidMount() {
+        const { pageName } = this.props;
+        API.fetchHelpikData(pageName, "en")
+            .then((response) => {
+                this.setState(() => ({
+                    helpikContent: response.data.text,
+                    helpikURL: response.data.url.replace("api.php", ""),
```

Comment:
Where does the `api.php` come from here? Can we not send it in the response from the server?